packages/ng-table-builder/src/components/ngx-header/ngx-header.component.ts

```diff
@@ -8,4 +11,13 @@ import { TableContentDirective } from '../../directives/table-content.directive'
     changeDetection: ChangeDetectionStrategy.OnPush,
     encapsulation: ViewEncapsulation.None
 })
-export class NgxHeaderComponent extends TableContentDirective {}
+export class NgxHeaderComponent extends TableContentDirective {
+    @Output() public readonly expandedChange: EventEmitter<boolean> = new EventEmitter<boolean>();
+    @AttributeBoolean() @Input() public expanded: InputBoolean = true;
```

Comment:
we really need to have `expanded` and `expandable` as input options? 
I think it’s confusing for me when I will try use this code

Reply:
the "expandable" field is responsible for whether the table can be expanded and expanded, and the "expanded" field is responsible for its state upon initialization.

Reply:
@Schirbak what do you think about this API?

Reply:
what do you think about

```html
<ngx-header expandablePanel>
```

```html
<ngx-header expandablePanel [expanded]="false">
```

and add `hideToggle` when whether the toggle indicator should be hidden.


```html
<ngx-header expandablePanel hideToggle>
```

Reply:
done